Makefile

```diff
@@ -23,6 +23,7 @@ SGX_DEBUG ?= 0
 SGX_PRERELEASE ?= 0
 SGX_PRODUCTION ?= 0
 
+SKIP_WASM_BUILD = 1
```

Comment:
I would put an `?=` here to have the option to build the WASM with use of setting an env variable

Reply:
In this case we never want to build wasm. This is to stop the substrate crates (maybe substratee-node-runtime only??, idk) from building a redundant wasm binary that is not needed.

Reply:
agree. the ? would just allow to build it in case wanted